src/config/ConfigService.js

```diff
@@ -116,7 +116,7 @@ class Config {
 
 
 	dispatch(store) {
-		var config = Object.assign({}, __CONFIG__, this._remote_config, this._defaults);
+		var config = Object.assign({}, __CONFIG__, this._defaults, this._dynamic_config);
```

Comment:
Hmms, this doesn't feel correct.

The sequence is:

Default, Static, dynamic.

So the order should be:

`this._defaults, __CONFIG__, this._dynamic_config`